src/core/response/parameters.rs

```diff
@@ -47,13 +57,80 @@ impl TryFrom<Json> for VpToken {
     type Error = Error;
 
     fn try_from(value: Json) -> Result<Self, Self::Error> {
-        serde_json::from_value(value).map(Self).map_err(Into::into)
+        match value {
+            // NOTE: When parsing a Json string object, it must be base64 encoded.
+            Json::String(s) => Ok(Self::Single(BASE64_STANDARD.encode(s.as_bytes()))),
```

Comment:
The spec says the value must be represented as a base64url JSON string, so you should be doing *base64url* instead of *base64*, and _decoding_ instead of _encoding_

Reply:
Commit [ee67e6b](https://github.com/spruceid/oid4vp-rs/pull/18/commits/ee67e6bb988c505ce7a73708fa0189975bd021a3) changes to `BASE64_URL_SAFE_NO_PAD`